add-matcha.php

```diff
@@ -0,0 +1,56 @@
+<?php
+//connect to the database and hide the db-connection.php into gitignore
+//get products from the database by creating a separate file for functions
+require_once 'db-connection.php';
+require_once 'functions.php';
+$db = getDb();
+
+if(isset($_POST['submit']))
+{
+    $image = $_POST['image'];
+    $productName = $_POST['product_name'];
+    $businessName = $_POST['business_name'];
+    $currency = $_POST['currency'];
+    $price = $_POST['price'];
+    $sql = "INSERT INTO products (image, product_name, business_name, currency, price)
+     VALUES ('$image', '$productName', '$businessName','$currency', '$price')";
```

Comment:
Probably want to bind params rather than parse the variables in for safety

Reply:
I will make this change in the next task!  I forgot to take them out in my experiment file!